src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <RootNamespace>System.Runtime.Serialization.Json.Tests</RootNamespace>
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
```

Comment:
Actually this are legacy tests that need to be supported on netcoreapp1.0, so if I upgrade this to 1.7 then we will only be able to run the tests in netcoreapp1.1. Do you think it's ok to leave this tests as 1.5 for now to keep them running (They are still testing ns1.7 because they have P2P references to csproj's) and when we remove the P2P references then change this to ns1.7(hopefully when we do that netcoreapp1.1 testing will be enabled)?